routes/chatRoutes.js

```diff
@@ -162,8 +162,36 @@ assistant:`;
 	console.log(`\n=====  REQUEST  =====\n${messagesToString([lastMessage])}`);
 
 	let stdoutStream = global.childProcess.stdout;
+	let stderrStream = global.childProcess.stderr;
 
-	const readable = new ReadableStream({
+	const stderr = new ReadableStream({
```

Comment:
what's the difference between stdout and steer? sorry not super well versed in this stuff lol

Reply:
stdout, stdin and stderr are the three text streams setup by the OS, available to any program to communicate with the terminal/console. llama just uses stderr to output a bunch of useful things, including errors, and we can potentially parse this for errors to return to clients for each invocation, although it's not used for such yet.